### ISSUE TITLE
unused arguments in wp_trend

### ISSUE THREAD

**Issue by erzk** [petermeissner/wikipediatrend#13]
I’ve installed the developmental version (on 03/05/2015) of wikipediatrend but I couldn’t run the code that worked in the CRAN version:
```
bkomo <- wp_trend(  page      = "Bronisław Komorowski", 
                  from      = "2015-01-01", 
                  lang      = "pl", 
                  friendly  = T, 
                  userAgent = T)
```
I was getting the following error:
```
Error in wp_trend(page = "Bronislaw Komorowski", from = "2015-01-01",  : 
  unused arguments (friendly = T, userAgent = T)
```
I removed the last two arguments of the code (friendly, userAgent) and it worked but still with no meaningful results (non-Latin characters issue that was filed earlier).

**Comment by petermeissner** [comment]
Thanks for reporting. From version 0.x to 1.x the parameters/options used by `wp_trend()` have changed.

I shall do some fixing for downward compatibility.

**Comment by petermeissner** [comment]
solved since version 1.1.0
